src/components/state-history-chart-line.js

```diff
@@ -311,6 +312,12 @@ class StateHistoryChartLine extends LocalizeMixin(PolymerElement) {
 
       // Concat two arrays
       Array.prototype.push.apply(datasets, data);
+
+      // Fire event to change dialog position
+      // Wait 500ms for the 0.3s transition to finish
+      setTimeout(() => {
+        fireEvent(this, "iron-resize");
```

Comment:
We should not have elements that might be in a dialog sometimes fire a resize event, especially not on every draw.